tests/ltp/buildenv.sh

```diff
@@ -45,6 +46,11 @@ if [[ "$mode" == "build" ]]; then
         git apply $FORK_DISABLE_PATCH
     fi
 
+    if [ -f $FIX_SETREUID_PATCH ];then
```

Comment:
This will not work since you are using same name for 04 and 07 patches. Please update both PRs and use FIX_SETREUID_PATCH04 and FIX_SETREUID_PATCH07 instead.(PR: https://github.com/lsds/sgx-lkl/pull/166)